src/toast/templates/template.py

```diff
@@ -45,14 +47,20 @@ class Template(TraitConfig):
     )
 
     det_data = Unicode(
-        None, allow_none=True, help="Observation detdata key for the timestream data"
+        defaults.det_data,
+        allow_none=True,
+        help="Observation detdata key for the timestream data",
     )
 
-    flags = Unicode(
-        None, allow_none=True, help="Observation detdata key for solver flags to use"
+    det_flags = Unicode(
+        defaults.det_flags,
+        allow_none=True,
+        help="Observation detdata key for solver flags to use",
     )
 
-    flag_mask = Int(0, help="Bit mask value for solver flags")
+    det_flag_mask = Int(
+        defaults.det_mask_invalid, help="Bit mask value for solver flags"
```

Comment:
Really nice to have det flags always enabled and a global definition of invalid data!